### DISCUSSION TITLE
Custom plugin location for composer based smarty package

### DISCUSSION THREAD

**Discussion by iamaks1993** [smarty-php/smarty#684] [Q&A] [answered]
Hi, 

I hope you guys are doing well. I have one question regarding smarty template. Currently i am using old smarty template which i have downloaded as folder. Then after i have created many plugins in it. As project grow ups we are adding all plugins through the composer. So my question is. Once we add smarty by using the composer then where we have to put our those plugins? So we don't need to do any manual work in the future for the moving a file. Thanks

**Comment by wisskid** [comment, marked as answer]
Please see https://smarty-php.github.io/smarty/programmers/api-functions/api-add-plugins-dir.html
